cmd/ceremony/cert.go

```diff
@@ -299,6 +317,8 @@ func makeTemplate(randReader io.Reader, profile *certProfile, pubKey []byte, ct
 		// it in our end-entity certificates.
 		cert.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth, x509.ExtKeyUsageServerAuth}
 		cert.MaxPathLenZero = true
+	case crossCert:
```

Comment:
I think we want to copy over the MaxPathLenZero from the tbcs as well -- our intermediates have it set, so cross-signs of our intermediates should have it set; our roots don't have it set, and cross-signs of our roots also don't have it set.